src/components/StudentFeedback.vue

```diff
@@ -51,14 +54,20 @@ export default {
   methods: {
     findAverage () {
       return 'Function placeholder'
-    },
-    async findModuleData () {
-      const response = await fetch(`https://shrouded-citadel-55795.herokuapp.com/api/v1/students/1/student_projects?mod=${this.module}`)
-      const data = await response.json()
-      this.moduleData = data
     }
+    // async findModuleData (module) {
+    //   const response = await fetch(`https://shrouded-citadel-55795.herokuapp.com/api/v1/students/1/student_projects?mod=${module}`)
+    //   this.moduleData = await response.json()
+    // }
   },
   created () {
+    fetch(`https://shrouded-citadel-55795.herokuapp.com/api/v1/students/1/student_projects?mod=${this.$route.params.id}`)
+      .then(response => response.json())
+      .then(data => {
+        console.log(data)
+        this.moduleData = data
+        this.projects = this.moduleData.data.attributes.student_projects
+      })
```

Comment:
I think I know why we had to move this fetch to `created`. The watch parameter was initially watching the route for changes, then calling the fetch request from methods, updating the state, and the state update was then triggering a re-render. By removing that watch and instead passing down a route parameter, we now need to make the fetch in `created`, since this lifecycle method triggers right after the initial build.